src/Client.cpp

```diff
@@ -321,9 +324,17 @@ bool Client::OnQuery(
     CEF_REQUIRE_UI_THREAD();
 
     if (frame->IsMain() && m_remoteTrigger) {
-        m_jobManager->Process(browser, 200);
-        callback->Success("OK");
-        return true;
+	if (request == "trigger") {
+            callback->Success("Processing");
```

Comment:
is this not prevented here? https://github.com/spajak/cef-pdf/pull/8/files/9e29e1405e178c2db8dc2de72d3170aff0136454..0f6d1dda739bef2c86cdc558bd9b982282fc229b#diff-ebaffda3ee7f99446ca2575b0a341c86R326